sandbox.sh

```diff
@@ -180,7 +180,7 @@ start_deku_cluster() {
   sleep 1
 
   echo "Producing a block"
-  HASH=$(deku-cli produce-block "$data_directory/0" | awk '{ print $2 }')
+  HASH=$(deku-cli produce-block "$data_directory/0" | tail -n 1 | cut -d ' ' -f 2)
```

Comment:
I suggest to use `sed -n 's/block.hash: \([a-f0-9]*\)/\1/p')` instead, more explicit and less fragile IMO.